docs/flashbots-auction/searchers/libraries/golang.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Golang library
+---
+
+Golang library is a repository that provides a high-level access to the `eth_sendBundle` and `eth_callBundle` rpc endpoint on [mev-relay](https://github.com/flashbots/mev-relay-js).
+
+Flashbots-enabled relays and miners expose two new jsonrpc endpoint: `eth_sendBundle` and `eth_callBundle`. Since these are brand-new, non-standard endpoints, ethers.js and other libraries do not natively support these requests (like `getTransactionCount`). In order to interact with these endpoints, you will also need access to another full-featured (non-Flashbots) endpoint for nonce-calculation, gas estimation, and transaction status.
+
+Access the Golang library repository [here](https://github.com/cryptoriums/flashbot).
```

Comment:
I'm checking something related to this :)

Reply:
sure let me know and I will apply all changes in one go.

Reply:
@obadiaa anything else you need me to do? I see the PR is approved, but let me know if it needs anything else

Reply:
Ping

Reply:
hey! apologies for the delay @krasi-georgiev!
i just added a little disclaimer that the library isn't maintained nor owned by flashbots for the readers, will look for your batch changes in one go and merge :)

Reply:
thanks, commited all your suggestions